src/css/components/_live.scss

```diff
@@ -1,11 +1,22 @@
-.video-js.vjs-live .vjs-time-control,
-.video-js.vjs-live .vjs-time-divider,
-.video-js.vjs-live .vjs-progress-control {
-  display: none;
+.vjs-live-control {
+  display: block;
+  font-size: 1em;
+  line-height: 3em;
 }
 
-.video-js .vjs-live-control {
+.vjs-live .vjs-time-control,
+.vjs-live .vjs-time-divider,
+.vjs-live .vjs-progress-control {
   display: none;
-  font-size: 1em;
-  line-height: 3em;
+}
+
+// Use the control spacer to fill in the middle space
+.vjs-live .vjs-custom-control-spacer {
+  @include display-flex();
```

Comment:
`updateShowing` doesn't control the visibility of the spacer, and the spacer is hidden by default so we still need this.